src/components/ModalBilling/ModalBilling.tsx

```diff
@@ -1,9 +1,13 @@
 import * as React from 'react';
 import classnames from 'classnames';
-import styles from './styles.module.scss';
+import {Elements} from '@stripe/react-stripe-js';
+import {loadStripe} from '@stripe/stripe-js';
 
+import styles from './styles.module.scss';
 import ModalPayment from '../ModalPayment';
 
+const stripePromise = loadStripe('pk_test_5AByIibLOhR6WHL3Mwnmel3P00zm0pIDrD');
```

Comment:
This key should also be updated to use `pk_test_5AByIibLOhR6WHL3Mwnmel3P00zm0pIDrD` for development, but to use our real key when deploying